### DISCUSSION TITLE
Random words capitalized

### DISCUSSION THREAD

**Discussion by djohnsonbhrp** [deepgram/community#505] [General help] [answered]
### Which Deepgram product are you using?

Deepgram SDKs

### Details

I just sent a test through and there are random words capitalized throughout several of the sentences in the paragraphs' texts.

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

e59a1546-f390-4b6a-9d07-be23d2e214e2

### If possible, please attach your code or paste it into the text box.

using System;
using System.Collections;
using System.Data;
using OutSystems.HubEdition.RuntimePlatform;
using OutSystems.RuntimePublic.Db;
using Deepgram.Clients;
using Deepgram.Models;
using Newtonsoft.Json;
using Deepgram;
using System.Threading.Tasks;

namespace OutSystems.NssDeepgram
{
    public class CssDeepgram : IssDeepgram
    {
        const string API_KEY = "my-api-key";

        public void MssTranscribeAudio(string ssFileUrl, out string ssTranscriptionText)
        {
            ssTranscriptionText = "";

            try
            {
                var credentials = new Credentials(API_KEY);
                var deepgramClient = new DeepgramClient(credentials);

                var response = Task.Run(async () =>
                    await deepgramClient.Transcription.Prerecorded.GetTranscriptionAsync(
                        new UrlSource(ssFileUrl),
                        new PrerecordedTranscriptionOptions()
                        {
                            Punctuate = true,
                            Utterances = true,
                            Diarize = true,
                            Model = "nova-2",
                            Version = "latest",
                            SmartFormat = true,
                            FillerWords = false,
                            Paragraphs = true,
                            Summarize = true,
                            MultiChannel = true,
                            DetectTopics = true
                        })).Result;

                ssTranscriptionText = JsonConvert.SerializeObject(response);
            }
            catch (Exception ex)
            {
                // Handle exceptions or log errors
                ssTranscriptionText = $"Error: {ex.Message}";
            }
        }
    }
}

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jjmaldonis** [comment]
Hi @djohnsonbhrp, thanks for the info and the request ID. I'll pass this on to our team as an additional example. We are currently working on a fix for intermittent capitalizations in sentences.

**Reply by djohnsonbhrp** [reply]
@jjmaldonis thanks. Will you post on here when that fix is live?

**Reply by jjmaldonis** [reply]
Yeah I'll let you know.

**Reply by jjmaldonis** [reply, marked as answer]
Hey @djohnsonbhrp, just letting you know we fixed the intermittent capitalization in sentences.
